Added templates for using FTP and SMB Connectors (#325)
wso2/ballerina-integrator: templates/ftp_listener/src/ftp_listener/Module.md

# FTP Listener

In this template we will implement an FTP listener which will listen to a remote directory and periodically notify the 
addition and deletion of files that comply with the specified file pattern.

## How to run the template

1. Alter the config file `src/ftp_listener/resources/ballerina.conf` according to your requirement. 

2. Execute the following command in the project directory `ftp_listener` to run the service.
```bash

> ballerina run ftp_listener

Reviewer: Add the path to the conf file as well.
--b7a.config.file=path/to/file

Author: Added.

```
3. Invoke the service by uploading a file with the same file name pattern defined in the config to the folder defined 

>    in the smb listener instance. Then the file is processed and added to the list along with its details. Invoke the 

Reviewer: ```suggestion
   in the ftp listener instance. Then the file is processed and added to the list along with its details. Invoke the 
```

Author: Fixed.

# FTP Listener

In this template we will implement an FTP listener which will listen to a remote directory and periodically notify the 
addition and deletion of files that comply with the specified file pattern.

## How to run the template

> 1. Alter the config file `src/ftp_listener/resources/ballerina.conf` according to your requirement. 

Reviewer: ```suggestion
1. Alter the config file `src/<module_name>/resources/ballerina.conf` according to your requirement. 
```

Author: Added.

2. Execute the following command in the project directory `ftp_listener` to run the service.
```bash

> ballerina run ftp_listener --b7a.config.file=src/ftp_listener/resources/ballerina.conf 

Reviewer: ```suggestion
ballerina run ftp_listener --b7a.config.file=src/<module_name>/resources/ballerina.conf 
```
Since the user can add any name for the module, i think its better not to specify this template module name.

Author: Added.
